src/hooks/useAccommodationsAndOffers/index.ts

```diff
@@ -161,9 +182,11 @@ export const useAccommodationsAndOffers = ({
   return {
     getOffersById,
     getAccommodationById,
+    allAccommodations,
     accommodations,
```

Comment:
`allAccommodations` is all unfiltered accommodations, `accommodations` are the final accommodations (filtered) and kept the name to prevent breaking several parts of the application, and I have no idea what `activeAccommodations` is or why it was created at all.

I would appreciate your recommendation on suitable names because I am honestly out of ideas on naming these fields.

Reply:
i see that `activeAccomodations` is not being used anymore, so we can remove it.

It was used to only display accomodations with offers ( so filtered accomodations), but somehow it was not being used anymore and a new variable was created for fitlering.